aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/wwwroot/view-resources/Views/Users/ChangePassword.js

```diff
@@ -0,0 +1,54 @@
+﻿(function ($) {
+    var _userService = abp.services.app.user,
+        l = abp.localization.getSource('AbpProjectName'),
+        _$form = $('#ChangePassword');
+
+    $.validator.addMethod("regex", function (value, element, regexpr) {
+        return regexpr.test(value);
+    }, l("PasswordsMustBeAtLeast8CharactersContainLowercaseUppercaseNumber"));
```

Comment:
Why do we force 8 length password ? We can remove the regex validation now and may implement https://docs.microsoft.com/en-us/aspnet/core/security/authentication/identity-configuration?view=aspnetcore-3.1#password in the future. @iyilm4z what do you think ?

Reply:
Angular uses same pattern,
https://github.com/aspnetboilerplate/module-zero-core-template/blob/adfd2bdd4dd69b136dccb9606158d83f152cdcf4/angular/src/app/users/change-password/change-password.component.ts#L51